static-instrumenter/build.gradle

```diff
@@ -0,0 +1,6 @@
+plugins {
+    id("otel.java-conventions")
```

Comment:
Still a couple more Gradle files in the PR to apply the other comments too

Reply:
Done :)